pkg/defaults/defaults.go

```diff
@@ -138,6 +140,12 @@ const (
 	DefaultGcpMachineType  = "n1-standard-1"
 	DefaultGcpRulePriority = 10
 
+	//default for VBox
+
+	DefaultVBoxVMName = "eve_live"
+	DefaultVBoxCpus   = 2
+	DefaultVBoxMemory = 4096
```

Comment:
Can we use existing constants here? Or we need separate default memory and CPUs count for VBox?

Reply:
Please address @giggsoff comments...